brave/brave-core
# Fixes publisher not showing in the panel if ac off

Resolves https://github.com/brave/brave-browser/issues/2084

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
- enable rewards
- disable ac
- visit verified site, unverified site and youtube video
- make sure that for all sites above correct data is shown in the panel

## Reviewer Checklist:
